src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -1072,6 +1077,8 @@ class DocBaseViewer extends BaseViewer {
             this.controls.add(__('toggle_findbar'), () => this.findBar.toggle(), 'bp-toggle-findbar-icon', ICON_SEARCH);
         }
 
+        this.pageControls.add(this.pdfViewer.currentPageNumber, this.pdfViewer.pagesCount);
```

Comment:
What as the reason for this moving?

Reply:
> What as the reason for this moving?

It's in the new design and I ask Matt, he says he does it on purpose.